system_tests/vision.py

```diff
@@ -392,3 +393,64 @@ def test_detect_safe_search_filename(self):
         self.assertEqual(len(safe_searches), 1)
         safe_search = safe_searches[0]
         self._assert_safe_search(safe_search)
+
+
+class TestVisionClientText(unittest.TestCase):
+    DESCRIPTIONS = (
+        'Do',
```

Comment:
I could limit the maxResults to 1 and do that but the API returns each word in order to provide it's position in the image. For instance if you have a sign or logo with stylized text in different locations, you might want to know if one of the words was in the right place.